scripts/Game/Monster/index.js

```diff
@@ -288,6 +280,18 @@ class Monster {
         pop();
         break;
     }
+    noStroke();
+    if (!this.burrowed) {
+      fill(0, 0);
+    } else {
+      fill(0);
+    }
+    ellipse(
+      this.position.x + camera.x,
+      this.position.y + camera.y,
+      this.size + 25,
+      this.size + 25
+    );
```

Comment:
You can just put the ellipse inside the if statement (and change it to this.burrowed, with the !), and remove the else.